indico/modules/events/contributions/models/fields.py

```diff
@@ -99,14 +99,13 @@ class ContributionField(db.Model):
     # - abstract_values (AbstractFieldValue.contribution_field)
     # - contribution_values (ContributionFieldValue.contribution_field)
 
-    @property
-    def field(self):
+    def get_field(self, management=False):
```

Comment:
This **completely** breaks the management of abstract fields elsewhere. This is the kind of change that requires a though check of every possible usage. 🌋